### DISCUSSION TITLE
Seal webpage

### DISCUSSION THREAD

**Discussion by BobbyESP** [JunkFood02/Seal#427] [Show and tell]
Well, today I've been working on a webpage for Seal. I found an opportunity for learning, so I decided to get a template from GitHub and I started creating a webpage. I'm leaving here a video if you want to see it. Thank you!

https://user-images.githubusercontent.com/60316747/207985061-5a47b8d2-00b4-495e-8ff3-5b38739b8188.mp4

**Comment by BobbyESP** [comment]
Well, it uses Fluent Design from Windows 11, that makes it a little confusing because lol, we are using Material 3 in the app.

**Comment by BobbyESP** [comment]
My IP is in there KEKW. Plz dont DDoS me 😆

**Comment by JunkFood02** [comment]
omg this looks so nice! thx for your hard work! 😳

**Reply by BobbyESP** [reply]
Haha thank you! I'm gonna polish it, make the READMEs better lol and later we can publish it if you want!

**Comment by BobbyESP** [comment]
https://user-images.githubusercontent.com/60316747/209801530-4025533d-d3e0-4667-ba7c-692b03920d22.mp4

**Reply by Marco-9456** [reply]
> 2022-12-28.11-55-11.mp4

This looks gorgeous ✨️✨️

**Reply by BobbyESP** [reply]
Haha thank you!

**Comment by HassanDalwi** [comment]
Where can we visit this webpage

**Reply by BobbyESP** [reply]
Not released, sorry :(
